docs/developer-guide/working-with-plugins.md

```diff
@@ -140,6 +140,11 @@ ruleTester.run("custom-plugin-rule", rule, {
 });
 ```
 
+#### Customizing RuleTester
+
+To create tests for each valid and invalid case, RuleTester internally uses `describe` and `it` methods from the Mocha test framework when it is available. If you use another test framework, you can override `RuleTester.describe` and `RuleTester.it` to make RuleTester compatible with it and have proper individual tests and feedback.
```

Comment:
Can you make RuleTester in code front everywhere? Seems best to keep it consistent.

Also, maybe show an example right here?